docker/drone/src/main/java/org/arquillian/cube/docker/drone/SeleniumContainers.java

```diff
@@ -25,6 +28,7 @@
     private static final int VNC_EXPOSED_PORT = 5900;
     public static final String SELENIUM_CONTAINER_NAME = "browser";
     public static final String VNC_CONTAINER_NAME = "vnc";
+    public static final String CONVERSION_CONTAINER_NAME = "flv2mp4";
```

Comment:
True, probably releasing the docker image should be a priority in before merging.